apko/private/apko_image.bzl

```diff
@@ -32,16 +32,29 @@ def _impl(ctx):
 
     args.add_all(ctx.attr.args)
 
+    lockfile = ctx.attr.contents[OutputGroupInfo].lockfile.to_list()[0]
```

Comment:
it's unfortunate that we have to call to_list here but i guess it's necessary as outputgroupinfo can only have depsets.

Reply:
Yes. And I assume it's cheap here... as it's only 1-element list always.